examples/ComplexExample/app/reducers/navigation.js

```diff
@@ -0,0 +1,5 @@
+import RootNavigator from "../navigators/root";
+
+export default function navigation(state, action) {
+  return RootNavigator.router.getStateForAction(action, state);
```

Comment:
This is not a bug, so you should change it to that. `getStateForAction` will return null when the state remains the same but the event was handled/swallowed.